crucible-mir/src/Mir/TransTy.hs

```diff
@@ -122,9 +122,6 @@ pattern CTyBvSize128 <- M.TyAdt _ $(M.explodedDefIdPat ["crucible", "bitvector",
 pattern CTyBvSize256 <- M.TyAdt _ $(M.explodedDefIdPat ["crucible", "bitvector", "_256"]) (M.Substs [])
 pattern CTyBvSize512 <- M.TyAdt _ $(M.explodedDefIdPat ["crucible", "bitvector", "_512"]) (M.Substs [])
 pattern CTyBv t <- M.TyAdt _ $(M.explodedDefIdPat ["crucible", "bitvector", "Bv"]) (M.Substs [t])
-pattern CTyBv128 <- CTyBv CTyBvSize128
-pattern CTyBv256 <- CTyBv CTyBvSize256
-pattern CTyBv512 <- CTyBv CTyBvSize512
```

Comment:
Worth noting the removal of these pattern synonyms in the `crucible-mir` changelog. (I doubt anyone is using them, but better safe than sorry.)